tests/conftest.py

```diff
@@ -2,12 +2,18 @@
 
 import pytest
 
-from jupyterhub_traefik_proxy import TraefikProxy
+from jupyterhub_traefik_proxy import TraefikEtcdProxy
 
 
 @pytest.fixture
-def proxy():
+async def proxy():
     """Fixture returning a configured Traefik Proxy"""
-    # TODO: set up the proxy
-    proxy = TraefikProxy()
+    proxy = TraefikEtcdProxy()
+    await proxy.start()
     yield proxy
```

Comment:
This should be:

```python
try:
    yield proxy
finally:
    await proxy.stop()
```

to ensure that the proxy stops whenever the fixture is used, without needing to call `proxy.stop` in all of the tests.